modules/ROOT/pages/values-and-types/temporal.adoc

```diff
@@ -897,6 +899,7 @@ Please note that:
 
 It is also possible to access the smaller (less significant) components of a component group bounded by the largest (most significant) component of the group:
 
+.Less significant components
```

Comment:
Shall we rename these to `nested components` optionally with `first/second order` if needed, and alter the surrounding text accordingly (and the rest of the doc)? I find the current naming/phrasing very hard to grasp.

Reply:
@JPryce-Aklundh

Reply:
Sounds good to me 👍

Reply:
Sounds good to me 👍